include/NovelRT.Interop/NrtInteropErrorHandlingInternal.h

```diff
@@ -14,10 +14,14 @@ void Nrt_setErrMsgIsAlreadyDeletedOrRemovedInternal();
 void Nrt_setErrMsgIsNotSupportedInternal();
 void Nrt_setErrMsgIsInitialisationFailureInternal();
 void Nrt_setErrMsgIsFunctionNotFoundInternal();
+void Nrt_setErrMsgIsNotInitializedInternal();
```

Comment:
Please note, this is a UK english codebase. Initialized needs to be spelt as Initialised.

Reply:
Ok